09-Multiparticle-Collision-Dynamics/02-Diffusion-in-Solution.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<p>I asked @clpetix to do it this way to demonstrate how to initialize an MPCD simulation from a GSD file. We might expect users to have made their initial configuration for the HOOMD particles in a GSD file elsewhere, but GSD cannot currently store MPCD particles so we cannot directly initialize from one. I hope we can add that ability later, but the current solution is to create a <code>hoomd.Snapshot</code> from the GSD file, then add the MPCD particles to the snapshot before initialization.</p><p> </p><p>Is it allowed to include a GSD file with a tutorial? That would make this intent clearer, as the code for creating the HOOMD particles is kind of boilerplate covered by other tutorials.</p>

Reply:
<p>Understood. I do demonstrate writing GSD files first in the other tutorials because this is a common usage pattern in production (initialization and simulation in separate files / methods). </p><p> </p><p>We should at least correct the reference to <code>Snapshot</code>.</p>

Reply:
OK! We can definitely clarify that this is a `Frame`.

Is it possible for us to include the premade GSD file itself with this tutorial (and omit the code that makes it from the tutorial), or do you prefer that the code for creating the GSD file lives in the notebook? If we keep the code, we should add a little discussion to make it clearer why we're doing it this way, as it probably seems unnecessarily complicated to someone who is already familiar with `hoomd.Snapshot`.

Reply:
We should keep the code. I find that users often download individual `.ipynb` files and run them. They get confused when there are other files required to run the notebook.

One solution might be to move the gsd file creation up into the "Boilerplate" section. Or perhaps to its own section.